src/Ilios/CliBundle/Command/InstallUserZeroCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Ilios\CliBundle\Command;
+
+use Ilios\CliBundle\Form\InstallUserZeroType;
+use Ilios\CoreBundle\Entity\Manager\AuthenticationManagerInterface;
+use Ilios\CoreBundle\Entity\Manager\SchoolManagerInterface;
+use Ilios\CoreBundle\Entity\Manager\UserManagerInterface;
+use Ilios\CoreBundle\Entity\Manager\UserRoleManagerInterface;
+
+use Matthias\SymfonyConsoleForm\Console\Helper\FormHelper;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Creates a first ("user zero") account with Course Director privileges.
+ *
+ * Class InstallUserZeroCommand
+ * @package Ilios\CoreBundle\Command
+ */
+class InstallUserZeroCommand extends ContainerAwareCommand
+{
+    /**
+     * @var string
+     */
+    const USERNAME = 'zero_user';
+
+    /**
+     * @var string
+     */
+    const PASSWORD = 'Ch4nge_m3';
+
+    /**
+     * @var string
+     */
+    const FIRST_NAME = 'User';
+
+    /**
+     * @var string
+     */
+    const LAST_NAME = 'Zero';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('form:install_user_zero')
+            ->setDescription('Creates a first user account with "Course Director" privileges.')
+            ->addOption(
+                'school',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'A valid school id.'
+            )
+            ->addOption(
+                'email',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'A valid email address.'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        /**
+         * @var UserManagerInterface $userManager
+         */
+        $userManager = $this->getContainer()->get('ilioscore.user.manager');
+
+        // prevent this command to run on a non-empty user store.
+        $existingUser = $userManager->findUserBy([]);
+        if (! empty($existingUser)) {
+            throw new \Exception('Sorry, at least one user record already exists. Cannot create a "user zero".');
+        }
+
+        /**
+         * @var SchoolManagerInterface $schoolManager
+         */
+        $schoolManager = $this->getContainer()->get('ilioscore.school.manager');
+
```

Comment:
the assumption is that this will run after the default data population has been loaded.
still in agreement that this should fail more gracefully...working on it.